rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -584,6 +584,17 @@ http {
 
     {{ end }}
 
+    # backend for when default-backend-service is not configured or it does not have endpoints
+    server {
+        listen {{ $all.ListenPorts.Default }} default_server {{ if $all.Cfg.ReusePort }}reuseport{{ end }} backlog={{ $all.BacklogSize }};
```

Comment:
I kind of repurposed this `$all.ListenPorts.Default`, I don't see it being used anywhere else.